Fixes

| ERROR: qtxmlpatterns-... do_populate_sysroot: QA Issue: Qt5XmlPatterns.pc failed sanity test (tmpdir) in path .../phytec/work/cortexa9hf-neon-phytec-linux-gnueabi/qtxmlpatterns/5.6.2+gitAUTOINC+074571305e-r0/sysroot-destdir/usr/lib/pkgconfig [pkgconfig]

Index: git/mkspecs/features/qt.prf
===================================================================
--- git.orig/mkspecs/features/qt.prf
+++ git/mkspecs/features/qt.prf
@@ -323,6 +323,16 @@ contains(TEMPLATE, .*lib) {
     pkgconfig_prefix_replace.CONFIG = path
     QMAKE_PKGCONFIG_INSTALL_REPLACE += pkgconfig_prefix_replace
 
+    pkgconfig_lib0_replace.match = "-L$$[QT_INSTALL_LIBS]"
+    pkgconfig_lib0_replace.replace = "-L\$${libdir}"
+    pkgconfig_lib0_replace.CONFIG = path
+    QMAKE_PKGCONFIG_INSTALL_REPLACE += pkgconfig_lib0_replace
+
+    pkgconfig_lib1_replace.match = "-L/usr/lib"
+    pkgconfig_lib1_replace.replace = "-L\$${libdir}"
+    pkgconfig_lib1_replace.CONFIG = path
+    QMAKE_PKGCONFIG_INSTALL_REPLACE += pkgconfig_lib1_replace
+
     # drop -L/usr/lib in .prl files
     prl_replace.match = "-L\$${libdir}"
     prl_replace.replace = ""
